sub.go

```diff
@@ -138,7 +145,24 @@ func (sub *subSocket) subscribe(topic string, v int) {
 	sub.mu.Unlock()
 }
 
+func (sub *subSocket) subscribed(topic string) bool {
+	sub.mu.RLock()
+	defer sub.mu.RUnlock()
+	if _, ok := sub.topics[""]; ok {
+		return true
+	}
+	if _, ok := sub.topics[topic]; ok {
+		return true
+	}
+	for k := range sub.topics {
+		if strings.HasPrefix(topic, k) {
+			return true
+		}
+	}
+	return false
+}
+
 var (
 	_ Socket = (*subSocket)(nil)
-	_ Topics = (*subSocket)(nil)
+	//_ Topics = (*subSocket)(nil)
```

Comment:
I'd rather keep this interface "contract".

Reply:
Ok, committed in issue-108.